### ISSUE TITLE
Unhandled NullReferenceException when reading LNURL

### ISSUE THREAD

**Issue by TindustrialMV** [LN-Zap/zap-android#204]
## Description
When scanning a withdrawal LNURL, that has no URL-Query data, the application crashes. Scanning a LNURL should never crash the Application.

## Expected Behavior
Scan a withdrawal LNURL. Withdrawal UI opens.

## Actual Behavior
Application crashes because of NullReferenceExeption, if LNURL has no Query

## Possible Fix
Check if the decoded LNURL has a Query before trying to use it. Class: LnUrlUtil Method: readLnUrl

NullReferenceException when using .contains on "null" query:
```
decodedUrl = new URL(decodedLnUrl);
if (decodedUrl.getQuery().contains("tag=login")) {
  listener.onError(ctx.getString(R.string.lnurl_unsupported_type), RefConstants.ERROR_DURATION_MEDIUM);
  return;
}
```

Fixed:
```
decodedUrl = new URL(decodedLnUrl);
if(decodedUrl.getQuery() != null) {
  if (decodedUrl.getQuery().contains("tag=login")) {
    listener.onError(ctx.getString(R.string.lnurl_unsupported_type), RefConstants.ERROR_DURATION_MEDIUM);
    return;
  }
}
```

## Steps to Reproduce
Scan a LNURL like:
LNURL1DP68GURN8GHJ7MRWVF5HGUEWVDHK6TMHD96XSERJV9MJ7CTSDYHHVVF0D3H82UNV9AT8V6T5F3V82JNJGFENX3MP0PF4XW2HX445X0FCVZX
Decoded:
https://lnbits.com/withdraw/api/v1/lnurl/VvitLXuJrBs3GaxSS9W5kC

**Comment by raphBTC** [comment]
Hey @TindustrialMV,

thanks for creating the issue and already digging up the problem.
This should definitely not happen.

**Comment by raphBTC** [comment]
Fix has been released in v0.3.1-beta
